layer2/oscal_generator.go

```diff
@@ -62,8 +62,8 @@ func (c *Catalog) ToOSCAL(controlHREF string) (oscal.Catalog, error) {
 					Ns:    "",
 					Parts: &[]oscal.Part{
 						{
-							ID:    ar.Id + ".R",
-							Name:  "recommendation",
+							ID:    fmt.Sprintf("%s_smt.%s_gdn", control.Id, ar.Id),
+							Name:  "guidance",
 							Ns:    oscalUtils.GemaraNamespace,
```

Comment:
The namespace can be removed as well since `guidance` is defined under the default OSCAL namespace.